CMakeLists.txt

```diff
@@ -202,19 +216,58 @@ endif (ENABLE_CLANG_AS_LIB)
 ##             TEST APP                      ##
 ###############################################
 
+## TestJit.cpp
+#----- Sources
+
+# It does make no sense building it if there is no LLVM, given that the JIT implementation relies on LLVM. 
+if (ENABLE_CLANG_AS_LIB)
+  set (VC_TESTJIT_APP_SRC TestJit.cpp)
+
+  set (VC_EXE_NAME libVC_testJit)
+
+  if ( CMAKE_BUILD_TYPE MATCHES DEBUG )
+    set_source_files_properties(${VC_TESTJIT_APP_SRC} PROPERTIES
+              COMPILE_FLAGS "-DVC_DEBUG -DHAVE_CLANG_AS_LIB -DFORCED_PATH_TO_TEST=\\\"${CMAKE_CURRENT_SOURCE_DIR}/test_code\\\""
+  )
+  else( CMAKE_BUILD_TYPE MATCHES DEBUG )
+    set_source_files_properties(${VC_TESTJIT_APP_SRC} PROPERTIES
+              COMPILE_FLAGS "-DHAVE_CLANG_AS_LIB -DFORCED_PATH_TO_TEST=\\\"${CMAKE_CURRENT_SOURCE_DIR}/test_code\\\""
+    )
+  endif( CMAKE_BUILD_TYPE MATCHES DEBUG )
```

Comment:
Can this be restructured to add a single flag for each set method? I think some can be grouped outside this `if`